Formula/qjackctl.rb

```diff
@@ -14,14 +14,23 @@ class Qjackctl < Formula
   depends_on "autoconf" => :build
   depends_on "qt5"
   depends_on "jack"
+  depends_on "portaudio" => :linked
+
+  # Fixes varaible length array error with LLVM/Clang
+  # Reported upstream: https://github.com/rncbc/qjackctl/issues/17
+  patch do
+    url "https://github.com/rncbc/qjackctl/pulls/18.patch"
```

Comment:
Oh, you're absolutely right, that must have been happened when changing to the short version suggested by `brew audit`. I'll fix it!